src/views/image_element.ts

```diff
@@ -76,7 +76,7 @@ export class ImageElementView extends BlockView<Trackable<ImageElementNode>> {
     this.subcontainer.classList.add('figure-block-group')
 
     this.contentDOM = document.createElement('div')
-    this.contentDOM.setAttribute('contenteditable', 'false')
+    this.contentDOM.setAttribute('contenteditable', 'true')
```

Comment:
there is another two set to `contenteditable` in this file we need to remove at all, but we need to run a regeression test to make sure we don't ruin figure and table